gulpfile.js

```diff
@@ -97,90 +97,28 @@ gulp.task('tsc:globals', function(done) {
 
 gulp.task('prepublish', function(done) {
   runSequence('checkVersion', ['jshint', 'format'], 'tsc', 'tsc:globals', 'types',
-    'ambient', 'built:copy', done);
+    'built:copy', done);
 });
 
 gulp.task('pretest', function(done) {
   runSequence('checkVersion',
     ['webdriver:update', 'jshint', 'format'], 'tsc', 'tsc:globals',
-    'types', 'ambient', 'built:copy', done);
+    'types', 'built:copy', done);
 });
 
 gulp.task('default',['prepublish']);
 
 gulp.task('types', function(done) {
-  var folder = 'built';
-  var files = ['browser', 'element', 'locators', 'expectedConditions',
-    'config', 'plugins', 'ptor'];
-  var outputFile = path.resolve(folder, 'index.d.ts');
+  var outputFile = path.resolve('built', 'index.d.ts');
```

Comment:
Instead of having a gulp task for this, can this now just be a `lib/index.ts` file?